### PR TITLE
Ingredients Details view and API

### PR DESCRIPTION
**Pull request by ssteph-sudo**

### CODE REVIEW

File: KnowBetter-WebApp/Controllers/IngredientsController.cs [hunk 1]
```diff
@@ -19,6 +24,21 @@ public IngredientsController(KnowBetter_WebAppContext context)
...
+        //    //{
+        //    //    return NotFound();
+        //    //}
+
+        //    var model = new APILinksModel();
+        //    string[] links = new string[] { "Link1", "Link2" };
+        //    model.IngredientName = name;
+        //    model.APILinks = links;
+        //    return View(model);
+        //}
```
Comment (moe-mohammed) [inline KnowBetter-WebApp/Controllers/IngredientsController.cs:39]: remove commented  code

File: KnowBetter-WebApp/Startup.cs [hunk 2]
```diff
@@ -29,6 +29,18 @@ public void ConfigureServices(IServiceCollection services)
...
+            //try
+            //{
+            //    var sp = services.BuildServiceProvider();
+            //    KnowBetter_WebAppContext context = sp.GetService<KnowBetter_WebAppContext>();
+            //    DataSeeder.SeedProducts(context);
+            //}
+            //catch (Exception e)
+            //{
+            //    throw new Exception("Database does not exist or is not properly initialized. Run 'Update-Database' from package manager console.", e);
+            //}
```
Comment (moe-mohammed) [inline KnowBetter-WebApp/Startup.cs:43]: remove unrelated changes

File: KnowBetter-WebApp/Models/APIResultModel.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using KnowBetter_WebApp.Models;
+
+namespace KnowBetter_WebApp.Models
+{
+    public class APIResultModel
+    {
+        public APIResults APIModel { get; set; }
```
[the remark this answers is not in the document]
Comment (moe-mohammed) [reply KnowBetter-WebApp/Models/APIResultModel.cs:11]: instead of creating a new class apiresults i mean, since the new class is just a dictionary

[the remark this answers is not in the document]
Comment (ssteph-sudo) [reply KnowBetter-WebApp/Models/APIResultModel.cs:11]: I realize what you are saying. I have chosen to do it this way for a reason. I feel we may need to add to this later. In fact, I have discussed with Josh today that I might be adding something to it. We can discuss further.

**Comment by ssteph-sudo** [comment]
I have pushed the changes you requested except the one I am not changing.

**Review by moe-mohammed — APPROVED** [review]
